.github/workflows/cypress.yml

```diff
@@ -60,13 +63,23 @@ jobs:
 
     steps:
       - name: Restore context
-        uses: actions/cache@v3
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
         with:
+          fail-on-cache-miss: true
           key: cypress-context-${{ github.run_id }}
-          path: /home/runner/work/viewer
+          path: ./
+
+      - name: Set up node ${{ needs.init.outputs.nodeVersion }}
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          cache: 'npm'
+          node-version: ${{ needs.init.outputs.nodeVersion }}
+
+      - name: Set up npm ${{ needs.init.outputs.npmVersion }}
+        run: npm i -g npm@"${{ needs.init.outputs.npmVersion }}"
```

Comment:
This is also important, restoring the context does NOT restore the npm/node version :see_no_evil: 
Meaning we've been running two different versions this whole time....